docs/development-guidelines.md

```diff
@@ -6,7 +6,9 @@ You need to have:
 
 - [rootless Docker](https://docs.docker.com/engine/security/rootless/)
   - Note that if Docker is running as root on you local machine, you may run into some permission issues. 
+  - Note that if you use Rootless mode, you might want to check that your Docker daemon is running properly, e.g., with `$ sudo systemctl status docker`. If your Docker daemon is inactive, you can start it manually using, e.g., with `$ sudo systemctl start docker`.  
```

Comment:
Is `sudo` necessary?

Reply:
Not for `$ sudo systemctl status docker`. I will remove it.